src/common/services/Firebase/GetData/PostUtils.tsx

```diff
@@ -11,3 +20,54 @@ export async function getPostInfo(postId: string): Promise<IpostData> {
   }
   throw new Error(`Post with provided ID: ${postId}, does not exist`);
 }
+
+// Do a better query once we have samples of posts
+export async function queryPosts(
```

Comment:
This function is OK if you meant to use it for testing, but we will not be able to use it in production since a user does not have access to all posts for a given community.

I explained this in detail in the Dabase Data Model Design Doc, Design Details section: https://www.notion.so/Database-Data-Model-Design-Doc-0dce3c84d0274e38a8f12dd3adcebf76